R/checkGamsSuccess.R

```diff
@@ -0,0 +1,21 @@
+#' Check whether Gams finished successfully
+#'
+#' Check which output file was written and derive state of Gams run
+#'
+#' If \code{output.gdx} was written, give success message.
+#' If \code{abort.gdx} was written or no output file exists, stop with error message.
+#'
+#' @author Ricarda Rosemann
+#'
+#' @param path character, path to search for gams output files
+#'
+checkGamsSuccess <- function(path) {
+  if (file.exists(file.path(path, "output.gdx"))) {
+    message("Gams was successful: \"output.gdx\" exists.")
+  } else if (file.exists(file.path(path, "abort.gdx"))) {
+    stop("Gams aborted with \"abort.gdx\" due to errors in at least one variable.")
```

Comment:
I'm not quite sure if we might want some reporting/analysis also based on aborted runs. We wouldn't get anything that is supposed to happen after this check as it throws an error. But I'm fine with going this way for now and maybe adapt in the future if we find something else useful.

Reply:
> I'm not quite sure if we might want some reporting/analysis also based on aborted runs. We wouldn't get anything that is supposed to happen after this check as it throws an error. But I'm fine with going this way for now and maybe adapt in the future if we find something else useful.

Right now reportMif requires ```output.gdx``` to exist - as long as this is the case, I think there should be an error if it's not there. I'm open to changing this in the future if we want to also analyse ```abort.gdx```.

Reply:
I agree